docs/import-from-whitehall.md

```diff
@@ -0,0 +1,110 @@
+
+```
+rake import:whitehall_migration["nuclear-decommissioning-authority","news_article","news_story,press_release"]
+```
+
+There is no limit to the number of subtypes (additional values are added comma separated).
+
+It is also possible to import a single document subtype (e.g. press releases) from a single publisher (e.g. NDA):
+
+```
+rake import:whitehall_migration["nuclear-decommissioning-authority","news_article","press_release"]
+```
+
+After running the rake task, a URL will be printed to the console which allows the status of import to be examined.
+
+## Running a migration locally using govuk-docker
+
+Using [govuk-docker](https://github.com/alphagov/govuk-docker), you can run an import locally without needing to replicate data.  In order to do this, you will need to override the [Plek](https://github.com/alphagov/plek) URL for Whitehall and Publishing API to point to staging.  Additionally, you will require the bearer tokens for these applications from [govuk-secrets](https://github.com/alphagov/govuk-secrets).
+
+### Getting bearer tokens
+
+Tokens can be obtained from govuk-secrets.  These allow access to the relevant remote applications without needing to enter a username and password.
+
+```
+cd govuk-secrets/puppet_aws
+bundle exec rake 'eyaml:edit[staging,apps]'
+```
+
+The values you need to obtain are:
+- `govuk::apps::content_publisher::whitehall_bearer_token` - the Whitehall bearer token
+- `govuk::apps::content_publisher::publishing_api_bearer_token` - the Publishing API bearer token
+
+### Setting up Sidekiq
+
+In order to run a migration locally, Sidekiq must be running when the rake tasks are executed.  As jobs are run independently of the rake task, Plek overrides and bearer tokens are also required here.
+
+```
+govuk-docker run -e PLEK_SERVICE_PUBLISHING_API_URI=https://publishing-api.staging.publishing.service.gov.uk -e PUBLISHING_API_BEARER_TOKEN=<> -e PLEK_SERVICE_WHITEHALL_ADMIN_URI=https://whitehall-admin.staging.publishing.service.gov.uk -e WHITEHALL_BEARER_TOKEN=<> content-publisher-worker bundle exec sidekiq -C config/sidekiq.yml
+```
+
+Replace `<>` with the bearer tokens you have previously obtained.
+
+### Running a migration
+
+Either a single document or multiple document import can be run, as detailed in previous sections.
+
+For example:
+
+```
+govuk-docker-run rake import:whitehall_migration["nuclear-decommissioning-authority","news_article"] env PLEK_SERVICE_PUBLISHING_API_URI=https://publishing-api.staging.publishing.service.gov.uk PUBLISHING_API_BEARER_TOKEN=<> PLEK_SERVICE_WHITEHALL_ADMIN_URI=https://whitehall-admin.staging.publishing.service.gov.uk WHITEHALL_BEARER_TOKEN=<>
+```
+
+Replace `<>` with the bearer tokens you have previously obtained.
+
+In order to access the URL printed at the end of the rake task, the Content Publisher application will need to be running locally:
+
+```
+govuk-docker-up
```

Comment:
This one is also directory specific could switch it to `govuk-docker up content-publisher-app`

Reply:
Changed this one too.